src/components/TransactionFailText/index.test.tsx

```diff
@@ -80,4 +81,17 @@ describe('TransactionFailText', () => {
 
     expect(() => screen.getByAltText('Info Tooltip')).toThrow()
   })
+
+  it('shows an error if the safe itself is connected', async () => {
+    const sel = require('src/routes/safe/container/selector')
+    ;(sel.sameAddressAsSafeSelector as jest.Mocked<unknown>) = jest.fn(() => true)
+    ;(sel.grantedSelector as jest.Mocked<unknown>) = jest.fn(() => true)
```

Comment:
`unknown` can be probably changed to `() => boolean`?

Reply:
Might be able to use `ReturnType<typeof sel.sameAddressAsSafeSelector>`, but I'm not sure.

Reply:
`ReturnType<typeof sel.sameAddressAsSafeSelector>` is any because `sel` is any. I'll go with `() => boolean`